Dockerfile.prod

```diff
@@ -1,17 +1,15 @@
-# syntax=docker/dockerfile:1
+# syntax=docker/dockerfile:1.9
 ARG PYTHON_VERSION=3.12
-ARG PLONE_VERSION
-FROM python:${PYTHON_VERSION}-slim-bookworm
 
-ARG PYTHON_VERSION
-ARG PLONE_VERSION
+FROM python:${PYTHON_VERSION}-slim-trixie
+
+SHELL ["sh", "-exc"]
 
 LABEL maintainer="Plone Community <dev@plone.org>" \
       org.label-schema.name="server-prod-conf" \
-      org.label-schema.description="Plone $PLONE_VERSION development image using Python $PYTHON_VERSION" \
+      org.label-schema.description="Plone production image using Python $PYTHON_VERSION and UV" \
```

Comment:
I don't see any uv in the prod-config image

Reply:
You are correct. There is no need to have uv here, but it makes sense to have an image following the same logic of being tagged with the Python version instead of the Plone version.